examples/grpc-bidirectional-streaming-copy/tests/grpc_bidirectional_streaming_test.bal

```diff
@@ -0,0 +1,41 @@
+// This is the Ballerina test for bidirectional streaming scenario.
+import ballerina/test;
+
+//Client endpoint configuration.
+ChatClient chatEp = check new("http://localhost:9090");
+
+@test:Config
+function testBidiStreamingService() returns error? {
+        // Executes the RPC call and receives the customized streaming client.
+        ChatStreamingClient streamingClient = check chatEp->chat();
+
+        // Reads the server responses in another strand.
+        future<error?> f1 = start readResponseTest(streamingClient);
+
+        // Sends multiple messages to the server.
```

Comment:
Are we having new empty line before each comment? Can we remove all empty new lines?

Reply:
@daneshk - Was this intentional?

Reply:
It is nor consistent. Some comments have empty new lines before them while some are not having

Reply:
What would be the best practice here? To keep the new lines or not? @daneshk @dilanSachi @anupama-pathirage

Reply:
Let's remove empty lines

Reply:
Done.